pkg/provisioning/cbnt/validation.go

```diff
@@ -0,0 +1,184 @@
+package cbnt
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest"
+
+	"github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest/bootpolicy"
+	"github.com/9elements/converged-security-suite/v2/pkg/intel/metadata/manifest/key"
+)
+
+type testObj struct {
+	image     []byte
+	kmpubkey  *manifest.Key
+	bpmpubkey *manifest.Key
+	km        *key.Manifest
+	bpm       *bootpolicy.Manifest
+}
+
+var (
+	image         []byte
+	bpmpubkeypath string
+	objUnderTest  testObj
+)
+
+func setObjUnderTest(imgpath, kmkeypath, bpmkeypath string) error {
```

Comment:
maybe it's just me, but I like to have my arguments explicitly typed like imgpath string, kmkeypath string.. and so forth.

Reply:
done